tests/python_package_test/test_dask.py

```diff
@@ -885,27 +885,27 @@ def test_model_and_local_version_are_picklable_whether_or_not_client_set_explici
                     assert_eq(preds_orig_local, preds_loaded_model_local)
 
 
-def test_find_open_port_works():
+def test_find_open_port_works(listen_port):
     worker_ip = '127.0.0.1'
     with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
-        s.bind((worker_ip, 12400))
+        s.bind((worker_ip, listen_port))
```

Comment:
> I think that test is too strict.

I think now test is too soft 😃 . What about checking some meaningful intervals? Something like
```
worker_ip = '127.0.0.1'
port = listen_port
with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
    s.bind((worker_ip, port))
    ...
assert port < new_port < port + 10
```

Reply:
ok I can change it to `port < new_port < port + 1000`, since that is the range that the function covers: https://github.com/microsoft/LightGBM/blob/5321fef67b5cc4711dd7b2de985d5fe2b76d90cf/python-package/lightgbm/dask.py#L72